src/modules/esl-utils/misc/uid.ts

```diff
@@ -1,6 +1,30 @@
-/** Generate unique id */
-export function generateUId(): string {
-  const fp = Date.now().toString(32);
-  const sp = Math.round(Math.random() * 1024 * 1024).toString(32);
-  return fp + '-' + sp;
-}
+const sequences = new Map<string, number>();
+
+/** Create and return sequential id */
+export const sequentialUID = (name: string, prefix: string = name) => {
+  const uid = (sequences.get(name) || 0) + 1;
+  sequences.set(name, uid);
+  return prefix + uid;
+};
+
+/** Reset {@link sequentialUID} generator */
+export const resetSequence = (name?: string) => {
```

Comment:
I think the name of the function is incomplete. It is hard to understand what is sequence will be reset. Maybe adding UID to the name will be enough.